src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -83,13 +83,13 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
   @Input() id: string = this._uniqueId;
 
   /** Used to specify the tabIndex value for the underlying input element. */
-  @Input() tabIndex: number = 0;
+  @Input() tabindex: number = 0;
```

Comment:
It should be `tabIndex`, as that's the name of the property on native elements. 
(e.g., with an input you'd do `<input [tabIndex]="...">`)